shared/shot.js

```diff
@@ -458,15 +458,6 @@ class AbstractShot {
     this._createdDate = val;
   }
 
```

Comment:
Should `this.favicon = attrs.favicon || null` (line 194) be removed also?

Reply:
Oh yeah, good catch.